README.md

```diff
@@ -6,19 +6,19 @@ Lead Maintainer: [Matt Edelman](https://github.com/grawk)
 
 Identify all locales under a given directory and call a passed in "writer" for each one
 
-## API
-
-`module.exports = function build(appRoot, writer, cb) {`
+## Options
 
 - `appRoot {String}` filesystem directory where `locales` directory resides. Under that would be structure e.g. `US/en`, `XC/zh`
-- `writer {Function}`
-  - `localeRoot {String}` root output directory for the given locale
+- `buildPath {String}` the directory relative to `appRoot` where compiled files will be placed, default: `.build`
+- `localesPath {String}` the directory relative to `appRoot` where locale files can be found, default: `locales`
```

Comment:
Currently, `makara-builder` is a package served exclusively with `makara-amdify` and `grunt-makara-amdify` which, in itself, reads from a [hard](https://github.com/krakenjs/makara-builder/blob/v2.x/index.js#L12) [coded](https://github.com/krakenjs/makara-builder/blob/v2.x/index.js#L17) `locales` directory and converts i18n `.property` files into JSON based on a `<country>/<language>` subdirectory structure.  Once parsing is complete it executes a result function (here individuated as `writer`) with parameters specifying the output directory based on the input directory `<language>-<country>` and `cb`.  The `writer` function must return a function with parameter (`out`) which will provide the resulting JSON.

Despite this being a somewhat overly voluptuous method of delivering JSONification of `.property` files I believe this to be useful for a personal project where I wish to specify the "input" and "output" directory for these files i18n files.  Currently these directories, [`.build`](https://github.com/krakenjs/makara-builder/blob/v2.x/index.js#L16) and `locales`, respectively, are harcoded.  Whilst this coincides with structure of krakenjs and its build files, unlike the rest of Grunt tasks which produce static files, their output is specified within `tasks/`.  `grunt-makara-amdify`, on the other hand, is simply included in [`Gruntfile.js`](https://github.com/krakenjs/generator-kraken/blob/master/app/templates/common/Gruntfile.js#L16) as a single, unconfigurable line.  

Thus this pull request highlights the lack of configuration of the module which can be used with wider array of functions which parse `.property` files.  Additionally, I hope it serves as a starting ground for including more "optionality" for `grunt-makara-amdify` and `makara-amdify` to be provided in krakenjs as this will allow other developers modify their structure as per their requirements (for instance, sub-kraken modules with grunt task to build them all?).